# Define the leg poses for left side...
leg_pose left_front
leg_pose left_middle
leg_pose left_back

# Define the leg poses for the right side...
leg_pose right_front
leg_pose right_middle
leg_pose right_back

# Define the neck's pose...
servo_pose neck


